packages/astro/src/core/build/internal.ts

```diff
@@ -84,6 +84,8 @@ export interface BuildInternals {
 	entryPoints: Map<RouteData, URL>;
 	ssrSplitEntryChunks: Map<string, Rollup.OutputChunk>;
 	componentMetadata: SSRResult['componentMetadata'];
+
+	middlewareEntryPoint?: URL;
```

Comment:
removed extra whitespace

```suggestion
	middlewareEntryPoint?: URL;
```